src/webapp/src/config.js

```diff
@@ -30,41 +30,49 @@ const ROOT_DIRS = ['./', DEFAULT_AUDIO_DIR];
 // was not done yet to maintain functionality. It's ok to change the command
 // object keys to arrays, but some downstream methods need to change as well
 const JUKEBOX_ACTIONS_MAP = {
-  // Command Aliases
-  // Player
-  play_music: {
-    commands: {
-      play_album: {},
-      play_folder: {},
-      play_single: {},
-    }
-  },
+    // Command Aliases
+    // Player
+    play_music: {
+        commands: {
+            play_album: {},
+            play_folder: {},
+            play_single: {},
+        }
+    },
+
+    // Audio & Volume
+    audio: {
+        commands: {
+            change_volume: {},
+            toggle_output: {}
+        },
+    },
```

Comment:
Let's try to avoid using 4 space indentation .. in JS we use 2 spaces.

Reply:
That was enforced by the editorconfig settings. I will correct it.
Maybe this should be included in editorconfig to not be changed again on accident?

Reply:
It probably should be